docs/en/connector-v2/source/Elasticsearch.md

```diff
@@ -25,7 +25,7 @@ support version >= 2.x and <= 8.x.
 | username                | string  | no       | -                 |
 | password                | string  | no       | -                 |
 | index                   | string  | yes      | -                 |
-| source                  | array   | no       | -                 |
+| source                  | array   | yes      | -                 |
```

Comment:
The `source` is not required. We can use `schema` too. Please refer https://github.com/apache/seatunnel/pull/7421/files#diff-8eef53f88ae098e1477f67d8053f946d3267578ec8e68fd472d22c298864f876R128. Could you please update the documentation to make sure users are aware of this behavior?

Reply:
<img width="870" alt="2024-08-17_101921" src="https://github.com/user-attachments/assets/eaf08e51-ad90-40b4-a433-0832c7ff07cc">
Is this description feasible?